docs/ref/models/expressions.txt

```diff
@@ -228,6 +228,22 @@ primary key value rather than a model instance::
     >> car.built_by
     3
 
+.. _using-f-to-sort-nulls-last:
+
+Using ``F()`` to sort nulls last
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``F()`` can be used to sort a model field with nulls last.
+The code below will sort the `last_contacted` nullable datetime field with nulls first::
```

Comment:
I believe the ordering will be undefined (or database dependent), not necessarily nulls first.